src/basic/string-util.h

```diff
@@ -236,7 +236,7 @@ static inline char* str_realloc(char *p) {
 char* string_erase(char *x);
 
 int string_truncate_lines(const char *s, size_t n_lines, char **ret);
-int string_extract_line(const char *s, size_t i, char **ret);
+extern int string_extract_line(const char *s, size_t i, char **ret);
```

Comment:
this shouldn't be necessary. last time i looked it's entirely redundant in C, no?

Reply:
yep, removed